playbooks/openshift-glusterfs/private/config.yml

```diff
@@ -33,6 +28,10 @@
       tasks_from: firewall.yml
     when:
     - openshift_storage_glusterfs_registry_is_native | default(True) | bool
+
+- name: Load kernel modules for nodes
+  hosts: oo_all_hosts
```

Comment:
That's a pretty wide net to cast, why not just oo_nodes_to_config?

Reply:
OH right, I forgot about that distinction. Yeah, that. Even better, is there a way to limit it to non-master nodes?

Reply:
Although I suppose we should account for weird configurations like single-node, so `oo_nodes_to_config` is probably better.

Reply:
Yea, I think any node is liable to mount of up gluster so it should be all nodes including master nodes.

Reply:
sorry guys, had unexpected PTO. @jarrpa we still need to this? you didn't do anything in parallel?

Reply:
Nothing on my end!

Reply:
k, will do this tomorrow. You should see fixed PR when you connect :)

Reply:
done